slick/src/main/scala/org/apache/pekko/stream/connectors/slick/javadsl/Slick.scala

```diff
@@ -54,6 +51,7 @@ object Slick {
       session: SlickSession,
       query: String,
       mapper: JFunction[SlickRow, T]): Source[T, NotUsed] = {
+
```

Comment:
nit: Random new line?

Reply:
just spreading out the code to make it more readable